examples/gitserver/README.md

```diff
@@ -38,6 +38,8 @@ Prerequisites:
     $ oc policy add-role-to-user edit -z git
     ```
 
+**NOTE:** To deploy the git server on OpenShift Online, you must switch to using persistent
+storage. See the *Persistent Storage* section below.
```

Comment:
Although I understand that https://github.com/openshift/origin/blob/master/examples/gitserver/gitserver.yaml is not a template like our other examples, would it still make sense to have an ephemeral and persistent volume version of this file for the gitserver.

If so, they couldn't the user just choose between the two versions like they do with our other examples?